packages/component-base/src/define.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * @license
+ * Copyright (c) 2021 - 2023 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+export interface CustomElementType extends CustomElementConstructor {
```

Comment:
This file should be `define.d.ts`, not `define.ts`.

Reply:
Fixed